src/components/questions/handle-key-press.property.spec.ts

```
@@ -0,0 +1,360 @@
+import { describe, it, expect, vi } from "vitest";
+import fc from "fast-check";
+import { handleKeyPress } from "./handle-key-press.ts";
+
+function createTextarea(value: string, selectionStart: number) {
+  const el = document.createElement("textarea");
+  el.value = value;
+  el.selectionStart = selectionStart as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)